koku/reporting/provider/ocp/models.py

```diff
@@ -268,6 +268,125 @@ class Meta:
     node_role = models.TextField(null=True)
 
 
+class OCPNetworkSummaryP(models.Model):
+    """Store summary of bytes in, bytes out, and costs per cluster per day"""
+
+    class PartitionInfo:
+        partition_type = "RANGE"
+        partition_cols = ["usage_start"]
+
+    class Meta:
+        db_table = "reporting_ocp_network_summary_p"
+        indexes = [models.Index(fields=["usage_start"], name="ocp_net_summ_usage_start")]
+
+    id = models.UUIDField(primary_key=True)
+
+    cluster_alias = models.TextField(null=True)
+    cluster_id = models.TextField()
+    # Cost model fields need to exist even though they are not relevant to this model
+    cost_model_cpu_cost = models.DecimalField(max_digits=33, decimal_places=15, null=True)
+    cost_model_memory_cost = models.DecimalField(max_digits=33, decimal_places=15, null=True)
+    cost_model_rate_type = models.TextField(null=True)
+    cost_model_volume_cost = models.DecimalField(max_digits=33, decimal_places=15, null=True)
+    data_source = models.CharField(max_length=64, null=True)
+    distributed_cost = models.DecimalField(max_digits=33, decimal_places=15, null=True)
+    infrastructure_data_in = models.DecimalField(max_digits=33, decimal_places=15, null=True)
+    infrastructure_data_out = models.DecimalField(max_digits=33, decimal_places=15, null=True)
+    infrastructure_markup_cost = models.DecimalField(max_digits=33, decimal_places=15, null=True)
+    infrastructure_raw_cost = models.DecimalField(max_digits=33, decimal_places=15, null=True)
+    raw_currency = models.TextField(null=True)
+    resource_count = models.IntegerField(null=True)
+    unit = models.CharField(max_length=63, null=True)
```

Comment:
Same as above, the `unit` field does not exist on the base table, we either need to add the `unit` to the base table and track it very specifically for networking bytes or remove the `unit` field and track the unit in the field name like we do with other fields in the OCP tables.

Reply:
How do we track the unit in the field name? Do we extract the units from the fileld name? That seems fragile.

Reply:
Well, if you look at a lot of the other fields in the OCP tables you'll see fields like `pod_effective_usage_cpu_core_hours` and `persistentvolumeclaim_capacity_gigabyte_months` which is what I meant by the unit is in the field name. It requires knowing and converting units before values are inserted into the field and then the unit for that field should be constant. It is definitely more fragile than tracking a unit, but its an alternative to adding a unit field to the tables if you wanted to go that route.